recipe/meta.yaml

```diff
@@ -7,14 +7,7 @@ package:
 
 source:
   url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
-  sha256: fc4de6feaa6682801abac19f8e4f3ed0ea56b39e7e5a89ab66c7e0a76f661445
-  patches:
-    # TODO: Remove this patch when 0.19 is released
-    - skip-tests-failing-on-0.18.2.patch
-    # See https://github.com/scikit-learn/scikit-learn/pull/9129
-    - backport_9129.patch
-    # https://github.com/scikit-learn/scikit-learn/pull/9311
-    - backport_9311.patch
```

Comment:
Given that we are no longer using these patches, am proposing that we drop the patch files as well in PR ( https://github.com/conda-forge/scikit-learn-feedstock/pull/62 ).